assets/js/modules/ads/pax/services.js

```diff
@@ -113,5 +116,20 @@ export function createPaxServices( registry, _global = global ) {
 		termsAndConditionsService: {
 			notify: async () => {},
 		},
+		partnerDateRangeService: {
+			// eslint-disable-next-line require-await
+			get: async () => {
```

Comment:
Does this actually _need_ to be an `async` function? If PAX won't error, I don't see why we can't omit the `async` here 🤔 

If we're going to leave an ESLint ignore in here though, which I'm fine with, we should add a comment explaining it.